docker/kubernetes.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+sudo apt install -y awscli  # Install the AWS CLI.
+aws --version               # Confirm the AWS CLI was installed.
+
+aws configure set a us-east-1
+aws configure set aws_access-key.id
+aws configure set aws_secret_access_key.key
+
+#install iam-authenticator
+curl -o aws-iam-authenticator https://amazon-eks.s3.us-west-2.amazonaws.com/1.17.7/2020-07-08/bin/linux/amd64/aws-iam-authenticator
```

Comment:
Why do we need aws-iam-authenticator when we have the config file?

Reply:
@RishabhJain2018 Config uses `get token` command from `iam-authenticator`